cardinal/ecs/ecb/tick.go

```diff
@@ -68,7 +69,10 @@ func (m *Manager) FinalizeTick() error {
 	if err = pipe.Incr(context.Background(), redisEndTickKey()).Err(); err != nil {
 		return eris.Wrap(err, "")
 	}
+	flushStartTime := time.Now()
 	_, err = pipe.Exec(ctx)
+	elapsedTime := time.Since(flushStartTime)
+	m.logger.Logger.Info().Int("flush_time", int(elapsedTime.Milliseconds())).Msg("redis_flush")
```

Comment:
should we make this `flush_time_ms`? just so its clear for people who are looking at logs and not the code